client/src/components/USBController.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+
+import useInterval from 'use-interval'
+
+import Button from './Button'
+import Text from './Text'
+import { isPresent, isMounted, doMount, doUnmount, isAvailable } from '../lib/usbstick'
+
+const USBController = () => {
+  const available = isAvailable()
+  const [present, setPresent] = useState(false)
+  const [mounted, setMounted] = useState(false)
+
+  useInterval(() => {
+    (async () => {
+      const p = await isPresent()
+      setPresent(p)
+      if (p)
+	setMounted(await isMounted())
+    })()
+  }, available ? 1000 : null);  
+  
+  if (!available) {
+    return (<React.Fragment></React.Fragment>)
```

Comment:
See: https://stackoverflow.com/a/58408887/101290
```suggestion
    return null
```